ui/analyse/src/study/studyCtrl.ts

```diff
@@ -463,6 +463,10 @@ export default class StudyCtrl {
       return false;
     }
     const componentCallbacks = () => {
+      if (!this.multiBoard.showResults())
+      {
+        this.ctrl.userJump(this.ctrl.initialPath)
```

Comment:
according to my tests, this flashes the current position, before reverting to the initial position.